go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/fortytw2/leaktest v1.3.0 // indirect
 	github.com/frankban/quicktest v1.7.3 // indirect
 	github.com/fsnotify/fsnotify v1.4.7
+	github.com/go-delve/delve v1.4.0 // indirect
```

Comment:
why is delve becoming a dependency?

Reply:
ah that is my bad, just looking for a way to run a particular unit test. removing.